### PR TITLE
Adapt to BlocksForm in Blocks Engine

### PR DESCRIPTION
**Pull request by nileshgulia1**
Related to #2207 .

### CODE REVIEW

**Comment by tiberiuichim** [comment]
@nileshgulia1 Merge with the current volto master, you have test snapshots that shouldn't be updated.

File: src/components/manage/Blocks/Block/Edit.jsx [hunk 1]
```diff
@@ -137,88 +122,60 @@ class Edit extends Component {
     const blockHasOwnFocusManagement =
       blocks.blocksConfig?.[type]?.['blockHasOwnFocusManagement'] || null;
 
-    return (
-      <div className={`ui drag block inner ${type}`}>
-        {Block !== null ? (
-          <div
-            role="presentation"
-            onClick={(e) => {
-              const isMultipleSelection = e.shiftKey || e.ctrlKey || e.metaKey;
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/Block/Edit.jsx:146]: There is code that was added to Volto after I've forked the blocks engine. This needs to be updated according to latest Volto master functionality. In particular here it's missing the multi-block copy/paste.

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

File: src/components/manage/Blocks/Block/Edit.jsx (hunk 1 above)
Comment (nileshgulia1) [reply src/components/manage/Blocks/Block/Edit.jsx]: Aha! Sorry I missed the changes from master Edit block. The test should pass now. Thanks

Comment (nileshgulia1) [reply src/components/manage/Blocks/Block/Edit.jsx]: and the diff is much smaller :)

**Comment by nileshgulia1** [comment]
@tiberiuichim Green now.

File: src/components/manage/Blocks/Block/Edit.jsx (hunk 1 above)
Comment (tiberiuichim) [reply src/components/manage/Blocks/Block/Edit.jsx]: Sorry to do this to you, but there's a few other places where you have to sync API changes. Basically everything in the blocks helpers needs to be compared with the code that you've ripped out in Form.jsx, as there has been updates in Form.jsx. 

https://github.com/plone/volto/blob/master/src/helpers/Blocks/Blocks.js

I think all changes in Form.jsx after September 2020 needs to be synchronized to the Blocks.js helper and BlockForm.

**Comment by tiberiuichim** [comment]
@nileshgulia1 It might be green, but I'm pretty sure that there's more changes to be integrated into Blocks.js helper.

**Review by tiberiuichim — APPROVED** [review]

**Review by sneridagh — APPROVED** [review]

**Pull request merged**